aiida_common_workflows/workflows/relax/vasp/generator.py

```diff
@@ -29,12 +43,12 @@ def __init__(self, *args, **kwargs):
 
     def _initialize_protocols(self):
         """Initialize the protocols class attribute by parsing them from the protocols configuration file."""
-        with open(str(pathlib.Path(__file__).parent / 'protocol.yml')) as handle:
+        with open(str(pathlib.Path(__file__).parent / 'protocol.yml'), encoding='UTF-8') as handle:
             self._protocols = yaml.safe_load(handle)
 
     def _initialize_potential_mapping(self):
         """Initialize the potential mapping from the potential_mapping configuration file."""
-        with open(str(pathlib.Path(__file__).parent / 'potential_mapping.yml')) as handle:
+        with open(str(pathlib.Path(__file__).parent / 'potential_mapping.yml'), encoding='UTF-8') as handle:
```

Comment:
Same comment as above. Also: if you just put both methods in the `__init__` method, I think the code becomes cleaner. Is there a reason to have these separate methods for the initialisation, i.e. are they also called outside the constructor?

Reply:
Same as above. We can clean this for sure, but somehow I would like to touch as little as possible on the codebase that will reproduce the ACWF stuff. More than happy to do major changes once we have a release version dedicated to that study. And notice that some important error settings (since we cannot pass this in the protocol for instance) is important for VASP for the ACWF, but is something that a user generally really should not play too much with. Hence, I would prefer that a release was made that is dedicated to the ACWF study.